tests/Doctrine/Tests/DBAL/Functional/Platform/QuotingTest.php

```diff
@@ -29,4 +32,41 @@ public static function stringLiteralProvider(): iterable
             'single-quote' => ["'"],
         ];
     }
+
+    /**
+     * @dataProvider identifierProvider
+     */
+    public function testQuoteIdentifier(string $identifier): void
+    {
+        $platform = $this->connection->getDatabasePlatform();
+
+        /**
+         * @link https://docs.oracle.com/cd/B19306_01/server.102/b14200/sql_elements008.htm
+         */
+        if ($platform instanceof OraclePlatform && $identifier === '"') {
+            self::markTestSkipped('Oracle does not support double quotes in identifiers');
+        }
+
+        $query = $platform->getDummySelectSQL(
+            'NULL AS ' . $platform->quoteIdentifier($identifier)
+        );
+
+        $row = $this->connection->fetchAssociative($query);
+
+        self::assertNotFalse($row);
+        self::assertSame($identifier, key($row));
+    }
+
+    /**
+     * @return iterable<string,array{0:string}>
```

Comment:
Let's omit numeric keys, shall we? Also, we should probably format this similarly to PHP code (with spaces after commas  and colons)

```suggestion
     * @return iterable<string, array{string}>
```

Reply:
Unless it's part of the standard, there's no way to keep it consistent across the codebase. There are existing code fragments formatted like in my PR. Fixing them here will just increase the inconsistency. I would start with the coding s standard proposal and disregard these issues until it's enforced by the standard.